packages/contracts/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_L1_MSG_SENDER =
```

Comment:
I thought this isn't the case anymore

Reply:
Hmmm maybe the variable name here is confusing. It's actually referring to the dummy value that the `L2CrossDomainMessenger` uses in order to save gas.

Reply:
I'll call it "dummy" and add a comment explaining this.